### PR TITLE
[gaudi] v35r0

### PR DESCRIPTION
**Pull request by vvolkl**
Overhaul of the cmake configuration now released.

### CODE REVIEW

File: var/spack/repos/builtin/packages/gaudi/package.py [hunk 1]
```diff
@@ -89,10 +86,23 @@ class Gaudi(CMakePackage):
...
+            self.define_from_variant("GAUDI_USE_XERCESC",         "optional"),
+            self.define_from_variant("GAUDI_USE_DOXYGEN",         "docs"),
+            self.define("GAUDI_USE_PYTHON_MAJOR",
+                        str(self.spec['python'].version.up_to(1))),
+            # todo:
+            self.define("GAUDI_USE_INTELAMPLIFIER",  False),
+            self.define("GAUDI_USE_GPERFTOOLS",      False), ]
+        # this is not really used in spack builds, but needs to be set
+        if self.spec.version < Version('34.99'):
+            args.append(["-DHOST_BINARY_TAG=x86_64-linux-gcc9-opt"])
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/gaudi/package.py]: You don't append lists, you append strings. You can remove the brackets here.

**Review by adamjstewart — CHANGES_REQUESTED** [review]
Want to be a maintainer for this package?

**Comment by vvolkl** [comment]
> Want to be a maintainer for this package?

Thanks for the review! I'm already listed as a maintainer :)

File: var/spack/repos/builtin/packages/gaudi/package.py (hunk 1 above)
Comment (vvolkl) [reply var/spack/repos/builtin/packages/gaudi/package.py]: Right, done!

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**